python/pythonlab/pythonlab_setup/pythonlab_setup/reset_neighborhood.py

```diff
@@ -0,0 +1,11 @@
+from neighborhood import World
+
+def reset_neighborhood():
+  """
+  Reset the neighborhood so the world will be re-generated on next painter initialization.
```

Comment:
Just confirming my understanding. So we reset the neighborhood every time we call on `setup_pythonlab` which is called on when we load `pythonlab`, and when we click on 'Run'. Is that correct?
For 'on next painter initialization', does this refer to when we initialize a [`Neighborhood` instance](https://github.com/code-dot-org/code-dot-org/blob/2839e7ce38b9d15e447697cb5b97a8046447a385/apps/src/codebridge/MiniAppPreview/NeighborhoodPreview.tsx#L34) (or when we initialize a `Painter`)?

Reply:
This is the next time a user calls `Painter()` in code